Changes.md

```diff
@@ -13,6 +13,7 @@ Improvements
 
 - LightTool : Changed spot light and quad light edge tool tip locations so that they follow the cone and edge during drag.
 - Arnold : Improved speed of translation of encapsulated scenes when using many threads.
+- OSL Expression : Added support for getting an element of a vector typed context variable using `context( variableName, index )` or `context( variableName, index, defaultValue )`. Negative indices can be used to get elements starting at the end of the array.
```

Comment:
Note to self : should be referring to `contextElement()` - sort out when updating Changes.md for release.